### ISSUE TITLE
Yeppp! - an extremely fast vector HPC library

### ISSUE THREAD

**Issue by dumblob** [daokoder/dao#505]
Recently I discovered a vector library http://www.yeppp.info/index.html (MIT licensed) which seems to be extremely efficient and multiplatform. Based on the fact, that Dao has built-in arrays, different math modules (e.g. Decimal) and bindings (e.g. DaoGSL), it might make sense to integrate also this fast library.

On the other hand, I didn't manually check if it's possible to statically link this library. Having such fast built-in math sounds appealing, but I'm still a little bit concerned about it's size.

In the worst case, we might make a simple optional module at some point (it should be fairly easy - see https://github.com/JuliaLang/Yeppp.jl).

**Comment by Night-walker** [comment]
I don't think it is a high-priority task. In its current state, DaoVM requires intensive work on fixing bugs and polishing the language's design. Adding more modules looks like procrastination for me when the language is half-usable and there is no real demand for that functionality to satisfy.

**Comment by daokoder** [comment]
I agree with @Night-walker, we should not divert efforts for this non-essential task. Also, I am not really convinced by Yeppp, when I looked into its repo and try to see some examples, I found few, which I found very strange given what I saw in its homepage. And it seems to mix different languages if I remember correctly.
